extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/authority/DummyCredentialsVerifier.java

```diff
@@ -14,11 +14,54 @@
 
 package org.eclipse.dataspaceconnector.registration.authority;
 
+import org.eclipse.dataspaceconnector.iam.did.spi.resolution.DidResolverRegistry;
+import org.eclipse.dataspaceconnector.identityhub.client.IdentityHubClient;
 import org.eclipse.dataspaceconnector.registration.authority.spi.CredentialsVerifier;
+import org.eclipse.dataspaceconnector.spi.monitor.Monitor;
+import org.eclipse.dataspaceconnector.spi.response.StatusResult;
 
+import static org.eclipse.dataspaceconnector.registration.utils.DidDocumentUtils.getIdentityHubBaseUrl;
+import static org.eclipse.dataspaceconnector.spi.response.ResponseStatus.ERROR_RETRY;
+import static org.eclipse.dataspaceconnector.spi.response.ResponseStatus.FATAL_ERROR;
+
+/**
+ * Implementation of {@link CredentialsVerifier} that only retrieves verifiable credentials,
+ * but performs no action on them. It authorizes any participant to onboard to the dataspace,
+ * as long as its Identity Hub can be accessed.
+ * <p>
+ * This is meant as a starting point for implementing custom dataspace onboarding logic.
+ */
 public class DummyCredentialsVerifier implements CredentialsVerifier {
```

Comment:
What is the reason not to use the `IdentityHubCredentialsVerifier` at this point?